### ISSUE TITLE
Support multiple XML input files

### ISSUE THREAD

**Issue by ymark** [isl/x3ml#27]
Support multiple XML input files (in a single run).
To ensure that the URIs will be given correctly apart from the XPATH use also an Identifier (could be just an index) of the XML input file

**Comment by ymark** [comment]
The new functionality has been implemented.
Briefly it merges all the xml input files into one DOM Document and parses it.

**Comment by ymark** [comment]
The functionality has been implemented in 9079a7f9

**Comment by ymark** [comment]
The functionality is also supported through the command line methods bb2c9f62
If the user wants to use multiple xml input file, then he can use the same parameter name (-xml) using a comma-separated list of xml input paths
```
java -jar x3ml-engine-exejar.jar -xml input1.xml,input2.xml,input3.xml ...
```
